pepper-apis/dsm-core/src/main/resources/logback.xml

```diff
@@ -16,8 +16,8 @@
         <filter class="ch.qos.logback.classic.filter.ThresholdFilter">
             <level>ERROR</level>
         </filter>
-        <encoder>
-            <pattern>%d{HH:mm:ss.SSS} %X{LogBreadCrumb} %X{X-Forwarded-For} C:%X{ClientId} S:%X{UserId} [%thread] %-5level %logger{36} %msg%n</pattern>
+        <encoder class="net.logstash.logback.encoder.LogstashEncoder">
```

Comment:
Won't this lose the client and user id from the logs?

Reply:
IP will come in via LB logs, but it would be nice to also preserve the forwarded-for header so IP is present the app logs as well.   The LogBreadCrumb has never been helpful, so that can probably go without incident.

Reply:
I can add those, I believe.
I mostly want to see if this works before investing in it further, but I can try to add those now if that makes the review process better.

Reply:
Totally fine to come back to them in a separate PR.  Let's just make sure this doesn't get to prod without them.

Reply:
BTW, on the DSM side at least those properties (ClientId, UserId) are never set, as far as I can tell.